app/client/cli/helpers/setup.go

```diff
@@ -18,6 +20,11 @@ func P2PDependenciesPreRunE(cmd *cobra.Command, _ []string) error {
 	// TECHDEBT: this is to keep backwards compatibility with localnet
 	flags.ConfigPath = runtime.GetEnv("CONFIG_PATH", "build/config/config.validator1.json")
 
+	// by this time, the config path should be set
+	configs.ParseConfig(flags.ConfigPath)
```

Comment:
Is this call just for side effects in viper?

I would've expected us to either do something with the config object that it returns (e.g. pass it to the runtime manager) OR use the runtime manager to load the same config file.

Reply:
> Is this call just for side effects in viper?

Yes. I unfortunately do not remember the exact reasoning but it was needed in relation to the changes I made in the Makefile

Reply:
(see: https://github.com/pokt-network/pocket/pull/891#discussion_r1263340347)

Reply:
#907